source/sxl_traffic_light_controller.rst

```diff
@@ -1188,26 +1878,28 @@ Traffic Counting: Number of vehicles of given classification |br|  |br| Used for
    :loc: H
    :spec: >{\raggedright\arraybackslash}p{0.15\linewidth} p{0.08\linewidth} p{0.13\linewidth} p{0.50\linewidth}
 
-   =========  ======  ============  ====================================================================================================================================================================
-   Name       Type    Value         Comment
-   =========  ======  ============  ====================================================================================================================================================================
-   starttime  string  [time stamp]  Time stamp for start of measuring. Format according to W3C XML dateTime with a resolution of 3 decimal places. All time stamps in UTC. E.g. 2009-10-02T14:34:34.341Z
-   P          long    [number]      Number of cars
-   PS         long    [number]      Number of cars with trailers
-   L          long    [number]      Number of trucks
-   LS         long    [number]      Number of trucks with trailers
-   B          long    [number]      Number of busses
-   SP         long    [number]      Number of trams
-   MC         long    [number]      Number of motor cycles
-   C          long    [number]      Number of bicycles
-   F          long    [number]      Number of pedestrians
-   =========  ======  ============  ====================================================================================================================================================================
+   =========  =======  ============  ====================================================================================================================================================================
+   Name       Type     Value         Comment
+   =========  =======  ============  ====================================================================================================================================================================
+   starttime  string   [time stamp]  Time stamp for start of measuring. Format according to W3C XML dateTime with a resolution of 3 decimal places. All time stamps in UTC. E.g. 2009-10-02T14:34:34.341Z
+   P          integer  [0-65535]     Number of cars
+   PS         integer  [0-65535]     Number of cars with trailers
+   L          integer  [0-65535]     Number of trucks
+   LS         integer  [0-65535]     Number of trucks with trailers
+   B          integer  [0-65535]     Number of busses
+   SP         integer  [0-65535]     Number of trams
+   MC         integer  [0-65535]     Number of motor cycles
+   C          integer  [0-65535]     Number of bicycles
+   F          integer  [0-65535]     Number of pedestrians
```

Comment:
Not sure what other max value you are refering to. But anyway.... The large max value is used because traffic counting value may need this. A common value used by traffic engineers is 1900 vehicles/hour as theoretical max flow in a single lane if the traffic is fully saturated.

I don't think it nececarily is any point with aligning this list with the one used for priority requests. This list is originally based on what can be profiled using traffic counting detectors (e.g. pneumatic tubes or inductive loop detectors).